apps/ledger-live-desktop/src/renderer/screens/platform/App.jsx

```diff
@@ -28,7 +29,13 @@ export default function PlatformApp({ match, appId: propsAppId }: Props) {
   const appId = propsAppId || match.params?.appId;
 
   const localManifest = useLocalLiveAppManifest(appId);
-  const remoteManifest = useRemoteLiveAppManifest(appId);
+  let remoteManifest = useRemoteLiveAppManifest(appId);
+
+  const buySellTestUrl = getEnv("BUY_SELL_TEST_URL");
+
+  if (buySellTestUrl) {
+    remoteManifest = { ...remoteManifest, url: buySellTestUrl };
+  }
```

Comment:
Can we use the same structure for mobile and ddl?

````
const remoteManifest = useRemoteLiveAppManifest(appId);

const manifest = localManifest || remoteManifest;

const buySellTestUrl = getEnv("BUY_SELL_TEST_URL");

  if (buySellTestUrl) {
    manifest.url = buySellTestUrl;
  }

Reply:
I suppose we could. Do you think it improves things or just keeps it consistent?